MCprep_addon/load_modules.py

```diff
@@ -47,10 +47,10 @@
 else:
 	from .materials import prep
 
-if "OptimizeScene" in locals():
-	importlib.reload(prep)
+if "optimize_scene" in locals():
+	importlib.reload(optimize_scene)
 else:
-	from .SceneOptimizer import OptimizeScene
+	from .MCprep_resources import optimize_scene
```

Comment:
Really sorry if I wasn't clear about this - I meant that it could read template blend files from MCprep_resources, but that I'd expect the `optimize_scene.py` file itself to live in the root `MCprep_addon` folder (one level up).